packages/ember-routing/lib/system/route.js

```diff
@@ -1979,6 +1985,10 @@ function buildRenderOptions(route, namePassed, name, options) {
     }
   }
 
+  if (options && indexOf(keys(options), 'outlet') !== -1 && typeof options.outlet === "undefined") {
+    throw new EmberError("You passed undefined as the outlet name.");
```

Comment:
Can we use the actual outlet name that the Dev tried to passed in the error message?